src/index.ts

```diff
@@ -137,7 +146,7 @@ export class GoogleToken {
     }
   }
 
-  private async getTokenAsync() {
+  private async getTokenAsync(): Promise<string|null|undefined> {
```

Comment:
Do `null` and `undefined` mean different things here?

Reply:
*sighs* it's a vestigial limb of the original API.  The original authors favored `null`, but `undefined` happens when a property isn't defined on the response.  I opened https://github.com/google/node-gtoken/issues/60 to track fixing this in a breaking way when we release 3.0.

Reply:
Ah, ok.